UPSTREAM_README.md

```diff
@@ -1,9 +1,27 @@
+![license](https://img.shields.io/badge/license-MIT-blue) [![pre-commit](https://img.shields.io/badge/pre--commit-enabled-yellow?logo=pre-commit&logoColor=white)](https://github.com/pre-commit/pre-commit) [![conventional-commits](https://img.shields.io/badge/conventional%20commits-1.0.0-yellow)](https://www.conventionalcommits.org/en/v1.0.0/) [![black](https://img.shields.io/badge/code%20style-black-000000)](https://github.com/psf/black) [![mypy](https://img.shields.io/badge/mypy-checked-brightgreen)](http://mypy-lang.org/) [![pylint](https://img.shields.io/badge/pylint-required%2010.0-brightgreen)](http://pylint.org/) [![pytest](https://img.shields.io/badge/pytest-enabled-brightgreen)](https://github.com/pytest-dev/pytest) [![coverage](https://img.shields.io/badge/coverage-required%20100%25-brightgreen)](https://github.com/nedbat/coveragepy)
 
 # Python Package Template
 
 This repository is intended to be a base template, a cookiecutter for a new Python package project while keeping [PEP518](https://www.python.org/dev/peps/pep-0518/) in mind. Because it’s hosted on Github it already utilizes a few [Github Actions](https://docs.github.com/en/actions) that enforce repository-side checks for continuous integration and that implement a semantic release setup. And while this package is a starting point for a Python project with good engineering practices, it’s intended to be improved and added to in various ways — see the [Wiki](https://github.com/jenstroeger/python-package-template/wiki) for more suggestions.
 
+## Table of Contents
+
+[Features](#features)  
+&emsp;[Typing](#typing)  
+&emsp;[Quality assurance](#quality-assurance)  
+&emsp;[Unit testing](#unit-testing)  
+&emsp;[Documentation](#documentation)  
+&emsp;[Versioning and publishing](#versioning-and-publishing)  
+&emsp;[Dependency analysis](#dependency-analysis)  
+&emsp;[Security analysis](#security-analysis)  
+&emsp;[Standalone](#standalone)  
+[How to use this repository](#how-to-use-this-repository)  
+[Git hooks](#git-hooks)  
+[Testing](#testing)  
+[Documentation](#documentation)  
```

Comment:
This `Documentation` heading duplicates one above, should be fixed.

Reply:
Commit https://github.com/jenstroeger/python-package-template/pull/102/commits/ec7f2d121d965a47c7215dc41949a9649ef5c8f5.